altair/vegalite/v5/display.py

```diff
@@ -86,6 +86,13 @@ def svg_renderer(spec: dict, **metadata) -> Dict[str, str]:
     )
 
 
+def render_jupyter(spec):
```

Comment:
Should we call this `jupyter_renderer` to be consistent with how the other renderer functions are named?

Reply:
Good call